docs/billing-integration.md

```diff
@@ -155,9 +155,14 @@ data:
     - name: servicecontrol
       servicecontrol:
         identity: gcp
-        # The service name is unique to your application and will be
-        # provided during onboarding.
-        serviceName: my-application.mp-my-company.appspot.com
+        # The service name is unique to your application. You can find 
```

Comment:
Update working on line 19 too?

Reply:
Do you mean line 159 ?

Reply:
@gibbleyg I have updated [line 19](https://github.com/linhlam-kc/marketplace-k8s-app-tools/blob/5b9604da37acf5f1e560a8656e892ecf3f940cda/docs/billing-integration.md?plain=1#L19) in the latest [commit](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/617/commits/4f9fc288009c18ff9ab1ab43424dc2fcee02d44a). Please check again and let me know if I need to update anything. Thanks !